README.md

```diff
@@ -0,0 +1,7 @@
+# Bridger [bridger]
+
+[![luacheck](https://github.com/v-rob/bridger/workflows/luacheck/badge.svg)](https://github.com/v-rob/bridger/actions)
+
+Adds a large number of advanced nodes conducive to building large, industrial bridges.
```

Comment:
```suggestion
Adds a large number of advanced nodes conducive to building large, industrial bridges to [Minetest](https://www.minetest.net).  Also see the [forum post](https://forum.minetest.net/viewtopic.php?t=18243).
```
So people can know from the readme where this originates from instead of having no context.